presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -3685,7 +3671,7 @@ private void assertOneNotNullResult(@Language("SQL") String query)
 
     private boolean insertOperationsSupported(HiveStorageFormat storageFormat)
     {
-        return storageFormat != HiveStorageFormat.DWRF;
+        return true;
```

Comment:
remove `insertOperationsSupported` method?

Reply:
Thanks, missed that